### PR TITLE
SOW-24: add phase banner

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
## Changes
- add phase banner component to gds-components
- show version banner under header on all pages

### CODE REVIEW

File: lib/gds-components/phase-banner/PhaseBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          <strong className="govuk-tag govuk-phase-banner__content__tag">
+            Beta
+          </strong>
+          <span className="govuk-phase-banner__text">
+            This is a new service
+          </span>
+          {linkUrl && (
+            <span>
+              – your{" "}
+              <a className="govuk-link" href={linkUrl}>
```
Comment (vferraro-scottlogic) [inline lib/gds-components/phase-banner/PhaseBanner.tsx:19]: TBC. Do we have a link for this?

Comment (vferraro-scottlogic) [reply lib/gds-components/phase-banner/PhaseBanner.tsx:19]: Yes. I havent seen the banner in any of the new designs except [in this sketch](https://www.figma.com/file/u68018Pmi1dkVYnqUBfrkZ/DLUHC-New-Local-Plans-timetable-dashboard-concepts?type=design&node-id=3571-4238&mode=design&t=zLcsBuCTe5kkB0Lp-0) which seems pretty old

File: lib/gds-components/phase-banner/PhaseBanner.tsx
[the remark this answers is not in the document]
Comment (vferraro-scottlogic) [reply lib/gds-components/phase-banner/PhaseBanner.tsx:1]: Done! :)

File: lib/gds-components/phase-banner/PhaseBanner.tsx (hunk 1 above)
Comment (Ro4052) [reply lib/gds-components/phase-banner/PhaseBanner.tsx:23]: I'll ask but for now, we can just get the banner in place

Comment (vferraro-scottlogic) [reply lib/gds-components/phase-banner/PhaseBanner.tsx]: I changed so that the text is displayed without link

Comment (vferraro-scottlogic) [reply lib/gds-components/phase-banner/PhaseBanner.tsx]: link removed completely

**Review by Ro4052 — APPROVED** [review]

**Pull request merged**